src/main/markdown/doc/latest/DevGuideI18nLocale.md

```diff
@@ -77,8 +77,11 @@ the following in your [module XML](DevGuideOrganizingProjects.html#DevGuideModul
 
 ```
 <set-property-fallback name="locale" value="en"/>
+<set-property name="locale" value="en"/>
```

Comment:
This should make it clear (with a comment) that the value should be the full list of supported locales, and that the goal is only to avoid compiling the `default` locale, that would never be used.